academy.test.js

```diff
@@ -0,0 +1,68 @@
+const { takeTurn, checkWinner, resetGame, getBoard } = require('./academy');
+
+
+const each = require("jest-each").default;
+
+describe("Test taking turns", () => {
+    //Arrange
+    each([
+        [//one turn
+            [[1,0]],
+            [[null,null,null],["nought",null,null],[null,null,null]]
+        ],
+        [//two turns
+            [[2,0],[2,2]],
+            [[null,null,null],[null,null,null],["nought",null,"cross"]]
+        ],
+        [//turn on top of another
+            [[1,1],[1,1]],
+            [[null,null,null],[null,"nought",null],[null,null,null]]
+        ]
+    ]).it("When the input is '%s'", (turns, expected_output) => {
+        //Act
+        for (let i=0; i<turns.length; i++) {
+            takeTurn(turns[i][0], turns[i][1])
+        }
+        actual_output = getBoard()
```

Comment:
Could pass the getBoard() function to the expect function (line 29) directly instead of assigning a variable but might reduce clarity.